layout/widget/google_adsense.ejs

```diff
@@ -0,0 +1,21 @@
+<div class="widget-wrap widget-float">
+<% if (theme.plugins.google_adsense) { %>
+    <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+    <!-- huemen sidebar -->
+    <ins class="adsbygoogle"
+            style="display:block"
+            data-ad-client="<%= theme.plugins.google_adsense %>"
+            data-ad-slot="9596486949"
```

Comment:
Is the slot ID always fixed?

Reply:
I'll work on an improvement to make this configurable.